content/backend/graphql-scala/0-introduction.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Introduction
+pageTitle: "Building a GraphQL Server with Scala Backend Tutorial"
+description: "Learn how to build a GraphQL server with Scala & Sangria and best practices for filters, authentication, pagination and subscriptions. "
+
+---
+
+### Motivation
+
+Scala is very popular language nowadays and it's often chosen to deliver efficient and distributed systems. It leverages the Java VM, known mostly for its efficiency. Support of Functional Programming and power of Java make able to deliver applications fast in either development or runtime.
```

Comment:
Few smaller remarks (pick the one you find useful)
very popular -> a very popular
efficient and distributed systems -> efficient and scalable systems (being distributed or not is a property of the solution/architecture, not language itself)
`known mostly for its efficiency` -> `known for its reliability and robustness`  
Support for Functional Programming, rich ecosystem and stable foundation allow building fast applications, quickly. 

Sorry to be picky on this paragraph, but it has to deliver a punch to keep the readers around